sklearn/metrics/_classification.py

```diff
@@ -2088,8 +2088,8 @@ def log_loss(y_true, y_pred, eps=1e-15, normalize=True, sample_weight=None,
 
     This is the loss function used in (multinomial) logistic regression
     and extensions of it such as neural networks, defined as the negative
-    log-likelihood of the true labels given a probabilistic classifier's
-    predictions. The log loss is only defined for two or more labels.
+    log-likelihood of the classfier given the true labels.
```

Comment:
I don't think this is helpful as a definition. I suppose we mean that `log_loss(y_true, y_pred)` is defined as the likelihood of a logistic model that returns `y_pred` probabilities for its training data `y_true`.

Reply:
We can definitely make it more specific.

Reply:
Incorporated in the pr.